client/my-sites/themes/themes-magic-search-card/style.scss

```diff
@@ -20,17 +20,15 @@
 	}
 
 	&.has-highlight {
-		box-shadow: 0 0 0 1px var( --color-primary ), 0 0 0 4px var( --color-primary-light );
-		position: relative;
-		z-index: z-index( 'root', '.themes-magic-search-card.has-highlight' );
```

Comment:
I think it might be possible to remove this too?

https://github.com/Automattic/wp-calypso/blob/782bae82a8ca57995cc6415226334a51e8156d98/client/assets/stylesheets/shared/functions/_z-index.scss#L87

Reply:
Good catch!